.github/workflows/github_server.yml

```diff
@@ -0,0 +1,83 @@
+name: GitHub server tests
+
+on:
+  push:
+  # why is workflow dispatch not showing up in github?
+  workflow_dispatch:
+    inputs:
+      tags:
+        required: False
+        description: 'Optional. Use a "tag expression" specify which tagged tests to run (https://cucumber.io/docs/cucumber/api/#tag-expressions)'
+        # # Only test 2 tests - one uses an env var, the other uses AL.
+        # # They test if env vars get set correctly and depencencies get
+        # # pulled in correctly.
+        # default: '@al1'
+      enable_tmate:
+        type: boolean
+        description: 'Run the build with tmate debugging enabled (https://github.com/marketplace/actions/debugging-with-tmate)'
+        required: false
+        default: false
+      # TODO: alkiln_version:
+
+jobs:
+
+  github-server-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    env:
+      # How would a dev make another user/password if they wanted to?
+      SECRET_VAR1: secret-var1-value
+      SECRET_VAR2: secret-var2-value
+      SECRET_FOR_MISSING_FIELD: secret for missing field
+      #_ALKILN_DANGEROUS_DEBUG: true
+      
+    steps:
+      # Place the root directory in this branch
+      - uses: actions/checkout@v3
+
+      # Debugging. Maybe add a long sleep after whatever it is we want to check
+      # to give us time to explore.
+      - name: Setup tmate session
+        if: ${{ inputs.enable_tmate }}
+        uses: mxschmitt/action-tmate@v3
+        with:
+          detached: true
```

Comment:
Is this something that we need to leave in?

Reply:
Need is a strong word, but if we leave it in it's easy to trigger it with a manual workflow to debug GitHub server type stuff. Maybe this another thing where we can make an issue and point to this code, though.

Reply:
Future People: Made and closed #802